mscore/musicxmlsupport.h

```diff
@@ -183,6 +183,8 @@ class MxmlSupport {
 
 class ValidatorMessageHandler : public QAbstractMessageHandler
       {
+      Q_OBJECT
```

Comment:
Hmm, seems the mtests don't like this Q_OBJECT here, but don't have any problem with the other one of this commit

Reply:
Thanks @dmitrio95 for the hint with adding a line to mtests/CMakeLists.txt...